roles/change_hostname/files/change_hostname.rb

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/ruby
+
+# BEFORE RUNNING THIS SCRIPT
+#
+# 1) log into UI, rename the default capsule under infrastructure->capsules to the new hostname in both the name and url fields
+#
+# 2) update NEW_HOSTNAME variable below
+#
+require "socket"
+
+STDOUT.puts "Starting the script run..."
+
+# update this variable
+NEW_HOSTNAME = ARGV[0]
```

Comment:
This script is currently called from inside the ansible role  `command: /tmp/change_hostname.rb {{ NEW_HOSTNAME }}`  and will not be changed by the user hopefully.  Do we still need this check?